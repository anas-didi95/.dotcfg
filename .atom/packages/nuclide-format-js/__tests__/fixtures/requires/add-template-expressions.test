`foo ${max(1,2,3)} bar`;
