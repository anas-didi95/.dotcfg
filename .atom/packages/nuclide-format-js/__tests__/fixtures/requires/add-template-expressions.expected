const max = require('max');

`foo ${max(1,2,3)} bar`;
